auctioneer/batch.go

```diff
@@ -22,9 +22,11 @@ var (
 
 // BatchSource abstracts the source of a trader's pending batch.
 type BatchSource interface {
-	// PendingBatch retrieves the ID and transaction of the current pending
-	// batch. If one does not exist, account.ErrNoPendingBatch is returned.
-	PendingBatch() (order.BatchID, *wire.MsgTx, error)
+	// PendingBatchSnapshot retrieves the snapshot and raw transaction of
+	// the currently pending batch. If there isn't one,
+	// account.ErrNoPendingBatch is returned.
+	PendingBatchSnapshot() (*clientdb.LocalBatchSnapshot, *wire.MsgTx,
```

Comment:
The batch transaction should already be in the snapshot so we can remove it from the return parameter.

Reply:
Right, done. Was also able to remove the pending batch TX key that was used for this.